src/rafts/lighthouse/commands/fractal.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable prettier/prettier */
+'use strict';
+/* We did this without ck's help */
+/* eslint-disable-next-line no-unused-vars */
+
+const { createCanvas } = require('canvas');
+
+const Discord = require('discord.js');
+
+const BaseCommand = require('../../BaseCommand');
+
+class FractalCommand extends BaseCommand {
+    constructor(boat) {
+        const options = {
+            name: 'fractal',
+            owner: true,
+            enabled: true,
+        };
+        super(boat, options);
+    }
+
+    run(message) {
+        const width = 1200;
+        const height = 1200;
```

Comment:
Maybe take these as options for the command

Reply:
atm no but maybe sometime in the future